p2p/src/net/libp2p/proto/connectivity.rs

```diff
@@ -286,7 +285,7 @@ mod tests {
 
     #[tokio::test]
     async fn outgoing_error_inbound_exists() {
-        let (mut backend, _, _, _, _) = util::make_libp2p(
+        let (_, _, _, _, _) = util::make_libp2p(
```

Comment:
This look is pretty much why returning big tuples is an anti-pattern. Please consider changing this to a struct some time in the future.